transforms/ember-object/__testfixtures__/inject-basic.input.js

```diff
@@ -0,0 +1,7 @@
+import { inject as controller } from '@ember/controller';
+import { inject as service } from '@ember/service';
```

Comment:
These were already tested in decorators.input.js but that file is pretty complex so I wanted something simpler to more easily detect if this particular thing breaks.